tools/ci/java-ci-tools/src/main/java/org/apache/flink/tools/ci/licensecheck/JarFileChecker.java

```diff
@@ -210,6 +210,11 @@ private static int findNonBinaryFilesContainingText(
                     // dual-licensed under GPL 2 and EPL 2.0
                     // contained in sql-avro-confluent-registry
                     .filter(path -> !pathStartsWith(path, "/org/glassfish/jersey/internal"))
+                    // contained in sql-connector-pulsar
+                    .filter(
+                            path ->
+                                    !pathStartsWith(
+                                            path, "/org/apache/pulsar/shade/org/glassfish/jersey/"))
```

Comment:
Pulsar client shades a lot of jars, such as the `avro`. Can we just change this check filter to `/org/apache/pulsar/shade`?

Reply:
This code is to skip checking license. Only `org/glassfish/jersey/` I find  needs to be skipped.